apps/web/components/booking/BookingListItem.tsx

```diff
@@ -307,14 +308,10 @@ function BookingListItem(booking: BookingItemProps) {
                           <RefreshIcon className="mr-1 -mt-1 inline-block h-4 w-4 text-gray-400" />
                           {`${t("every_for_freq", {
                             freq: t(
-                              `${RRuleFrequency[booking.eventType.recurringEvent.freq]
-                                .toString()
-                                .toLowerCase()}`
+                              `${Frequency[booking.eventType.recurringEvent.freq].toString().toLowerCase()}`
                             ),
                           })} ${booking.recurringCount} ${t(
-                            `${RRuleFrequency[booking.eventType.recurringEvent.freq]
-                              .toString()
-                              .toLowerCase()}`,
+                            `${Frequency[booking.eventType.recurringEvent.freq].toString().toLowerCase()}`,
```

Comment:
This piece of very long and repeated string throughout the app will be refactored in an util as part of #2750